docs/docs/howto/cluster-setup/bare-metal.md

```diff
@@ -62,12 +62,25 @@ fs.inotify.max_user_instances = 8192
 
 Apply this change by running `systemctl restart systemd-sysctl` and verify it using `sysctl fs.inotify.max_user_instances`.
 
-## K3s setup
+## Kubernetes cluster setup
+
+Contrast can be deployed with different Kubernetes distributions.
+
+### K3s
+
+[K3s](https://k3s.io/) is a lightweight Kubernetes distribution that's easy to set up.
```

Comment:
```suggestion
[K3s](https://k3s.io/) is a lightweight Kubernetes distribution that is easy to set up.
```
nit: avoid colloquial contractions in written documents like docs

Reply:
This is required by the microsoft style guide we are following, and also enforced by ci.

Reply:
Oh, wait, really? Interesting.